packages/spectacle/src/components/fullscreen.tsx

```diff
@@ -19,6 +19,14 @@ const Container = styled('div')`
 const FullScreen = forwardRef<HTMLDivElement, FSProps>(
   ({ size, color, ...props }, ref) => {
     const toggleFullScreen = useToggleFullScreen();
+    const [isClient, setIsClient] = useState(false);
+
+    useEffect(() => {
+      setIsClient(true);
+    }, []);
```

Comment:
This component requires `document` to exist, so we need to only render it client side.